io_scene_dae/export_dae.py

```diff
@@ -211,14 +203,11 @@ def export_image(self, image):
                 image.filepath = img_tmp_path
 
         else:
-            # Export relative, always, no one wants absolute paths.
             try:
-                # Export unix compatible always
                 imgpath = os.path.relpath(
                     imgpath, os.path.dirname(self.path)).replace("\\", "/")
-
             except:
-                # Fails sometimes, not sure why
+                # TODO: Review
```

Comment:
It's inside an `except` block, so I thought it was clear what the problem is. Readding it, in any case.